src/locales/en_US/en_US.json

```diff
@@ -245,6 +245,11 @@
         "button_secondary": "Preview video",
         "title": "Check your video"
     },
+    "doc_multi_frame_capture": {
+        "instructions_title_front": "Position the front of your document in the frame",
+        "instructions_title_back": "Position the back of your document in the frame",
+        "capture_progress_title": "Scanning document"
```

Comment:
Have these been added to the lokalise dashboard? Are you planning to support other languages?

Reply:
Please add a list of new keys to the migration file

Reply:
This feature is still ongoing and translations keys are likely to change soon.